src/common/Routes.ts

```diff
@@ -1,9 +1,8 @@
 export enum RoutePaths {
-  DASHBOARD = "/dashboard",
-  HABIT_TRACKER = "/habit-tracker",
-  RECOMMENDATIONS = '/recommendations',
-  START_QUESTIONNAIRE = '/start-questionnaire',
-  QUESTIONNAIRE = '/questionnaire',
-  LOGIN = '/login',
-  LANDING = '/landing'
+  DASHBOARD = "/OralCheckr/dashboard",
+  HABIT_TRACKER = "/OralCheckr/habit-tracker",
+  RECOMMENDATIONS = '/OralCheckr/recommendations',
+  START_QUESTIONNAIRE = '/OralCheckr/start-questionnaire',
+  QUESTIONNAIRE = '/OralCheckr/questionnaire',
+  LOGIN = '/OralCheckr/login'
```

Comment:
we want to avoid repetition. What if your path changes in the future? you will have to make a change to each and every route.
You can do this:

```
const BASE_PATH = "/OralCheckr"

export const getFullPath = (route: string) => `${BASE_PATH}${route}`

export enum RoutePaths {
  DASHBOARD = "/dashboard",
  HABIT_TRACKER = "/habit-tracker"
}
```

and in the Router.tsx, you can do: `getFullPath(Routes.DASHBOARD)`

Reply:
All routes are updated to use getFullPath